.github/workflows/build_wheels.yml

```diff
@@ -61,7 +58,10 @@ jobs:
         CIBW_ARCHS_LINUX: x86_64
         CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
         CIBW_SKIP: "*musllinux*"
-        CIBW_BEFORE_ALL_LINUX: "yum install -y libuuid-devel"
+        CIBW_BEFORE_ALL_WINDOWS: "git config --global url.https://github.com/.insteadOf git://github.com/"
+        CIBW_BEFORE_ALL_LINUX: >
+          "yum install -y libuuid-devel" &&
+          "git config --global url.https://github.com/.insteadOf git://github.com/"
```

Comment:
I think we need a `CIBW_BEFORE_ALL_MACOS` with the `git config` command here, too